src/main/java/carracing/domain/entity/Participant.java

```diff
@@ -0,0 +1,30 @@
+package carracing.domain.entity;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+
+public class Participant {
+
+  private static final String COMMA = ",";
+
+  private final List<String> participant;
+
+  public Participant(List<String> participant) {
+    this.participant = participant;
+  }
+
+  public static Participant parse(String names) {
+    return new Participant(Arrays.stream(names.split(COMMA))
+                                 .collect(toList()));
```

Comment:
의미있는 중요한 로직은 의미있는 이름의 메서드로 분리해주면 가독성이 높아집니다. 

```suggestion
    return new Participant(getSplittedNameByDelimiter(COMMA))
```